pages/dr3/files.rst

```diff
@@ -546,69 +546,9 @@ the colors.
-5. Click "Search".
-6. For DECaLS only images, refine the search by Proposal ID (2014B-0404) in the "Refine" tab.
-7. The Results page offers several different ways to download the data. See `the Tutorials page`_ for details.
-
-.. _`Astro Data Lab's Science Archive`: http://archive.noao.edu/search/query
-.. _`the Tutorials page`: http://archive.noao.edu/tutorials/query
-
-
-(ii) ftp sites
---------------
-
-Following the organization of the Stacked images, Raw and Calibrated DECam images are organized
-by survey brick, which are defined in the file *survey-bricks-dr3.fits.gz* for DR3. Both the main Tractor
-catalogs and Sweep catalogs include the ``BRICKNAME`` keyword (corresponding to ``<brick>`` with
-format ``<AAAa>c<BBB>)``.
-
-- Raw: ftp://archive.noao.edu/public/hlsp/decals/dr3/raw/``<AAA>/<brick>``
-- Calibrated: ftp://archive.noao.edu/public/hlsp/decals/dr3/calibrated/``<AAA>/<brick>``
-- Stacked: ftp://archive.noao.edu/public/hlsp/decals/dr3/coadd/``<AAA>/<brick>``
-
-For the calibrated images, filenames can be retrieved from the ``IMAGE_FILENAME`` keyword in each brick
-from *legacysurvey*-``<brick>``-*ccds.fits*. Additionally, each *calibrated*/``<AAA>/<brick>``
-contains an ASCII file
-with a list of ``EXPID`` and ``IMAGE_FILENAME``
-(*legacysurvey*-``<brick>``-*image_filename.txt*; see the example below).
-``EXPID`` contains the exposure number and the CCD name (Nxx or Sxx) with the format ``EXPNUM-ccd``.
-There is one entry per CCD. Often, multiple CCDs from a given file are used so there are
-fewer unique filenames than the number of CCDs. Each *legacysurvey*-``<brick>``-*image_filename.txt*
-file contains the number of unique images in the last row (File Count).
-
-For the Raw CCD images, the file naming convention has evolved during the survey. The
-corresponding files can be reconciled through the original DECam filename:
-DECam_<``EXPNUM``>.fits.fz where ``EXPNUM`` needs to be in format ``I08`` and can be retrieved
-from *legacysurvey*-``<brick>``-*ccds.fits* for each brick, and from the keyword ``DTNSANAM``
-in ``hdr[0]`` from each calibrated file.
-
-Here is an example ASCII file for a given brick: *[noao-ftp]/calibrated/006/0060p147/legacysurvey-0060p147-image_filename.txt*
-
-::
-
-   expid                                                image_filename
-   1 00483709-N25 decam/CP20151010/c4d_151011_041055_oki_g_v1.fits.fz
-   2 00483709-N26 decam/CP20151010/c4d_151011_041055_oki_g_v1.fits.fz
-   3 00483709-N29 decam/CP20151010/c4d_151011_041055_oki_g_v1.fits.fz
-   4 00483710-N25 decam/CP20151010/c4d_151011_041329_oki_r_v1.fits.fz
-   5 00483710-N26 decam/CP20151010/c4d_151011_041329_oki_r_v1.fits.fz
-   6 00483710-N29 decam/CP20151010/c4d_151011_041329_oki_r_v1.fits.fz
-   File Count: 2
-
-In the example above, there are 6 CCDs used for the stacked image, corresponding to 2 unique, multi-extension files.
+See the `raw data page`_.
```

Comment:
My preference would be to just completely remove these "See the `raw data page`" entries (one on each of the dr1-dr9 files pages). I think it's a good idea to have this in a generic drop-down menu instead, and I think it would be a clean break to simply not mention the raw data page outside of that menu.

I suppose there's a backward-compatibility argument for leaving links in place, though, so, it's your call.

Reply:
Yes, it's already being added to the 'Other Data' drop down menu.  I think I'll leave the existing stubs in place though, so at least people won't wonder why the section disappeared entirely.